wonkyung/app.js

```diff
@@ -0,0 +1,44 @@
+const http = require("http");
+const express = require("express");
+const cors = require("cors");
+const morgan = require("morgan");
+const dotenv = require("dotenv");
+
+dotenv.config();
+
+const { DataSource } = require('typeorm')
+
+const myDataSource = new DataSource({
+  type: process.env.TYPEORM_CONNECTION,
+  host: process.env.TYPEORM_HOST,
+  port: process.env.TYPEORM_PORT,
+  username: process.env.TYPEORM_USERNAME,
+  password: process.env.TYPEORM_PASSWORD,
+  database: process.env.TYPEORM_DATABASE
+})
+
+myDataSource.initialize()
+  .then(() => {
+      console.log("Data Source has been initialized!")
+  });
+
+
+const app =express()
+
+app.use(express.json());
+app.use(cors());
+app.use(morgan(''));
+
+
+app.get("/ping", (req,res) => {
+  res.json({message:"pong"});
+});
+
+const server = http.createServer(app)
+const PORT =  process.env.PORT;
+
+const start = async () =>{
+   server.listen(PORT, () => console.log(`server is listening on ${PORT}` ));
+}
```

Comment:
express 객체를 app 이라는 객체로 받아서 사용하기 때문에 서버를 작동시키는 함수를 아래와 같이 수정해 주세요!
```suggestion
const start = async () =>{
   app.listen(PORT, () => console.log(`server is listening on ${PORT}` ));
}
```